cli/server.go

```diff
@@ -897,31 +897,37 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 			}
 
 			// Manage notifications.
-			cfg := options.DeploymentValues.Notifications
-			metrics := notifications.NewMetrics(options.PrometheusRegistry)
-			helpers := templateHelpers(options)
+			var (
+				cfg                  = options.DeploymentValues.Notifications
```

Comment:
Nit: This variable name is too generic without being limited by function scope.

Reply:
This was the existing name, but I'm happy to change it as I agree with you.